marklogic-data-hub/src/main/java/com/marklogic/hub/impl/FlowManagerImpl.java

```diff
@@ -48,7 +48,7 @@
 public class FlowManagerImpl extends ResourceManager implements FlowManager {
     private static final String HUB_NS = "http://marklogic.com/data-hub";
     private static final String NAME = "ml:flow";
-
+t
     private DatabaseClient stagingClient;
```

Comment:
There is a compile error here.

Reply:
oh yes of course.  id found that in my editor, but i guess that was after it pushed.  ill fix momentarily

Reply:
resolved